docs/BigQueryTable-batchsource.md

```diff
@@ -39,6 +39,10 @@ Datasets are top-level containers that are used to organize and control access t
 Each record is composed of columns (also called fields).
 Every table is defined by a schema that describes the column names, data types, and other information.
 
+**Partition From**: Partition start date. This value is ignored if the table does not support partitioning.
```

Comment:
Doc should mention how this is expected to be specified with an example. Timestamp? String with a specific format?

Reply:
Is the start date inclusive? exclusive?

Reply:
These names should match the labels used in the widget, which has 'Partition From Date'.

Reply:
Done.